src/ui/templates/results/resultsheader.hbs

```diff
@@ -15,13 +15,10 @@
     {{#if customResultsCount}}
       {{{customResultsCount}}}
     {{else}}
-      <div class="yxt-ResultsHeader-resultsCount" aria-label="{{resultsCountStart}} through {{resultsCountEnd}} of {{resultsCount}}">
+      <div class="yxt-ResultsHeader-resultsCount"
+           aria-label="{{translate phrase='{{start}} through {{end}} of {{resultsCount}}' start=resultsCountStart end=resultsCountEnd resultsCount=resultsCount }}">
         <div aria-hidden="true">
-          <span class="yxt-ResultsHeader-resultsCountStart">{{resultsCountStart}}</span>
-          <span class="yxt-ResultsHeader-resultsCountDash">-</span>
-          <span class="yxt-ResultsHeader-resultsCountEnd">{{resultsCountEnd}}</span>
-          <span class="yxt-ResultsHeader-resultsCountOf">of</span>
-          <span class="yxt-ResultsHeader-resultsCountTotal">{{resultsCount}}</span>
+          {{translate phrase='<span class="yxt-ResultsHeader-resultsCountStart">{{start}}</span><span class="yxt-ResultsHeader-resultsCountDash">-</span><span class="yxt-ResultsHeader-resultsCountEnd">{{end}}</span><span class="yxt-ResultsHeader-resultsCountOf">of</span><span class="yxt-ResultsHeader-resultsCountTotal">{{resultsCount}}</span>' context='Example: 1-10 of 50' start=resultsCountStart end=resultsCountEnd resultsCount=resultsCount }}
```

Comment:
Again, I believe these should be brackets around the interpolation placeholders

Reply:
Hey Connor! For code reviews in our repos, when a reviewer leaves a comment, the author of the PR should address the issue throughout the entire PR. There's no need to comment on each occurrence!